docs/writing-rules/pattern-syntax.md

```diff
@@ -139,14 +139,28 @@ The ellipsis operator can match any number of arguments to binary operations. Th
 foo = 1 + 2 + 3 + 4
 ```
 
-## Arrays
+## Containers
```

Comment:
Note if there’s anything pointing to the previous header at `pattern-syntax/#arrays` (such as a CLI link) then you’ll want to add a manual html anchor link with the `#arrays` name.

Reply:
Nothing that I'm aware of!